helm-chart/binderhub/templates/secret.yaml

```diff
@@ -6,7 +6,7 @@ type: Opaque
 data:
   {{- $values :=  dict "config" dict }}
   {{- $cfg := .Values.config }}
-  binder.hub-token: {{ .Values.jupyterhub.hub.services.binder.apiToken | b64enc | quote }}
+  binder.hub-token: {{ .Values.jupyterhub.hub.services.binder.apiToken | required "jupyterhub.hub.services.binder.apiToken must be explicitly set!" | b64enc | quote }}
```

Comment:
Note that this would just error when b64enc receives a null value with a less obvious error message, so adding this `required` pipe isn't a breaking change.